templates/systemd/funkwhale-celery-beat.service.j2

```diff
@@ -11,7 +11,8 @@ ExecStart={{ devture_systemd_docker_base_host_command_docker }} exec {{ funkwhal
     celery \
     --app=funkwhale_api.taskapp beat \
     --loglevel={{ funkwhale_loglevel }} \
-    -s /tmp/celerybeat-schedule
+    -s /tmp/celerybeat-schedule \
+    {{ funkwhale_celery_beat_process_extra_arguments | join(' ') }}
```

Comment:
If `funkwhale_celery_beat_process_extra_arguments` is an empty list, this will yield incorrect syntax. Example:

```
-s /tmp/celerybeat-schedule \
```

For this reason, other roles which inject "process extra arguments" do this injection somewhere in the middle.

Something like this could work nicely in all cases :

```
    celery \
    --app=funkwhale_api.taskapp beat \
    --loglevel={{ funkwhale_loglevel }} \
    -s /tmp/celerybeat-schedule {{ funkwhale_celery_beat_process_extra_arguments | join(' ') }}
```